src/plugins/intel_gna/serial/headers/2dot4/gna_model_header.hpp

```diff
@@ -129,7 +129,7 @@ struct RuntimeEndPoint {
                     void* descriptor_ptr,
```

Comment:
do we need `scaleFactor` to be double?
CC @nkogteva

Reply:
> do we need `scaleFactor` to be double? CC @nkogteva

In header scale is float

Reply:
@mryzhov BTW I can't find where this constructor is used